src/Makefile.test.include

```diff
@@ -364,8 +364,8 @@ endif
 if TARGET_WINDOWS
 else
 if ENABLE_BENCH
-	@echo "Running bench/bench_bitcoin ..."
-	$(BENCH_BINARY) > /dev/null
+	@echo "Running bench/bench_bitcoin (one iteration sanity check)..."
```

Comment:
Also added a comment to preempt "why are we running the benchmarks in make check at all" questions.